modules/op-approving-task-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-approving-task-web-console_{context}"]
+= Approving a manual approval task by using the web console
+
+You can approve or reject a manual approval task by using the {OCP} web console.
+
+If you are listed as an approver in a manual approval task and a pipeline run reaches this task, the web console a notification. You can view a list of tasks that require your approval and approve or reject these tasks.
+
+.Prerequisites
+
+* You enabled the {pipelines-shortname} console plugin.
+
+.Procedure
+
+. View a list of tasks that you can approve by completing one of the following actions:
+** When a notification about a task requiring your approval displays, click *Go to Approvals tab* in this notification.
+** In the *Administrator* perspective menu, select *Pipelines* -> *Pipelines* and then click the *Approvals* tab.
```

Comment:
Just double checking, though it's probably correct - is the UI really making you select Pipelines twice in a row? Ignore this comment if that's the case

Reply:
Yes it does, thanks for the double check